security/s2a/internal/crypter/common.go

```diff
@@ -0,0 +1,29 @@
+package crypter
+
+import "errors"
+
+const (
+	// GcmTagSize is the tag size - the difference in length between plaintext
+	// and ciphertext. From crypto/cipher/gcm.go in the Go crypto library.
+	GcmTagSize = 16
+	// NonceSize is the size of the nonce in number of bytes.
+	NonceSize = 12
+)
+
+// ErrAuth is the error produced on authentication failure.
```

Comment:
Can you make the comment more precise? This is the error produced by the AEAD crypter when something happens.

Reply:
I just removed this and added the message directly in `Decrypt` since that is the only place where it is used. I had it initially since it was present in ALTS.